SECP-service/src/main/resources/assets/app/index.html

```diff
@@ -32,9 +32,13 @@
                 <!-- Collect the nav links, forms, and other content for toggling -->
                 <div class="collapse navbar-collapse navbar-ex1-collapse">
                     <ul class="nav navbar-nav">
-                        <li>
+                        <li ng-cloak ng-show="!isAuthenticated">
                             <a href="/login">Login</a>
                         </li>
+
+                        <li ng-cloak ng-show="isAuthenticated" ng-click="logout()">
+                            <a href="/">Logout</a>
+                        </li>
                     </ul>
```

Comment:
@lavatahir is working on the profile page, when he is done we can add a profile icon to link to a users profile page when authenicated in the navbar